Makefile

```diff
@@ -74,10 +76,19 @@ endif
 	@printf "The engine has been cleaned.\n"
 
 version: variables
+ifneq ($(OVERWRITE_MOD_VERSION),"True")
```

Comment:
Can you please put these checks around https://github.com/OpenRA/OpenRAModSDK/blob/master/packaging/linux/buildpackage.sh#L69 (and likewise in the other platform scripts) instead of nooping out the behaviour here?